mingw-w64-libgpr/PKGBUILD

```
@@ -3,7 +3,7 @@
 _realname=libgpr
 pkgbase=mingw-w64-${_realname}
 pkgname=${MINGW_PACKAGE_PREFIX}-${_realname}
-pkgver=23.0.0
+pkgver=24.0.0
 pkgrel=2
```

```suggestion
pkgrel=1
```

Please reset the pkgrel to 1 when updating the version.